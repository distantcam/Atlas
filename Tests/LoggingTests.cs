using System.Linq;
using ApprovalTests.Reporters;
using NUnit.Framework;

[TestFixture]
[UseReporter(typeof(DiffReporter))]
public class LoggingTests
{
    [Test]
    public void InfoMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Infos.OrderBy(s => s), "Info");
    }

    [Test]
    public void WarningMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Warnings.OrderBy(s => s), "Warning");
    }

    [Test]
    public void ErrorMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Errors.OrderBy(s => s), "Error");
    }
}