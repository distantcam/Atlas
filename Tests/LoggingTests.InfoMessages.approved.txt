Info[0] = Creating mapping method for 'MissingMatchesSource' in mapping type 'MissingMatchesDestination'.
Info[1] = Creating mapping method for 'SimpleSource' in mapping type 'SimpleDestination'.
Info[2] = Creating mapping type for 'MissingMatchesDestination'.
Info[3] = Creating mapping type for 'SimpleDestination'.
Info[4] = Removing reference to 'Atlas'.
